contrib/sync_db_pools/codegen/src/lib.rs

```diff
@@ -41,6 +41,11 @@ use proc_macro::TokenStream;
 ///
 ///     Retrieves a connection wrapper from the configured pool. Returns `Some`
 ///     as long as `Self::fairing()` has been attached.
+//!
+//!   * `async fn run<R: Send + Static>(&self, impl FnOnce(&mut Db) -> R) -> R`
```

Comment:
How did you determine this method signature? It doesn't match the generated signature and is missing a few bounds.

Reply:
My bad, I relied on https://github.com/SergioBenitez/Rocket/blob/5a4e66ec439411d30f16e5c045f8e4986f5883a4/contrib/sync_db_pools/codegen/src/database.rs#L104-L107 but have incorrectly attempted to simplify the signature (i.e. thought that `FnOnce(&mut X) -> R + Send + 'static`'s `Send + 'static` relate to `R` rather than `F`).

Reply:
Attempted to fix this in cea7264